1/backend/src/resolvers.js

```diff
@@ -0,0 +1,86 @@
+const jwt = require('jsonwebtoken');
+const fs = require('fs');
+
+data = require('./data');
+
+let SECRET_KEY = fs.readFileSync('./src/key/secret.key', 'utf8');
+
+const resolvers = {
+    Query: {
+        todos: () => JSON.parse(JSON.stringify(data.todos))
```

Comment:
You're trying to make a clone of `data.todos`? Use [`lodash`](https://lodash.com/) if you really need to clone but I think here it's fine to do:
```js
        todos: () => data.todos
```